dstep/main.d

```diff
@@ -59,8 +59,9 @@ auto parseCLI (string[] args)
         std.getopt.config.passThrough,
         std.getopt.config.caseSensitive,
         "output|o", &config.output,
-        "language|x", &parseLanguage,
         "objective-c", &forceObjectiveC,
+        "language|x", "Treat subsequent input files as having type <language>.",
```

Comment:
Aha, I see. Would it be better to move it down to custom entries: https://github.com/Dicebot/dstep/blob/9a720e5e3a20ec4ad6a61437525d96d256892fd8/dstep/main.d#L227. Otherwise this will be spread across places.